bazel/cargo/Cargo.Bazel.lock

```diff
@@ -42,77 +37,10 @@ version = "0.4.22"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "a7a70ba024b9dc04c27ea2f0c0548feb474ec5c54bba33a7f72f873a39d07b24"
 
-[[package]]
-name = "once_cell"
-version = "1.19.0"
-source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "3fdb12b2476b595f9358c5161aa467c2438859caa136dec86c26fdd2efe17b92"
-
-[[package]]
-name = "proc-macro2"
-version = "1.0.86"
-source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "5e719e8df665df0d1c8fbfd238015744736151d4445ec0836b8e628aae103b77"
-dependencies = [
- "unicode-ident",
-]
-
 [[package]]
 name = "proxy-wasm"
 version = "0.2.2"
 dependencies = [
  "hashbrown",
  "log",
 ]
-
-[[package]]
```

Comment:
I take it these are just automated updates due to changes in hashbrown dependencies?

Reply:
Correct, and the CI enforces that as well.